src/main/java/frc/robot/DriveBase.java

```diff
@@ -703,37 +793,66 @@ public static class FollowTrajectory extends CommandBase {
                 new PIDController(0, 0, 0), // Right controller (usually the same values as left controller)
                 // vvv this is supposed to be type BiConsumer<Double, Double>
                 this.drivebase::setDriveVoltage, // Voltage biconsumer
-                true, // Should the path be automatically mirrored depending on alliance color. Optional, defaults to true
+                true, // Sh(ould the path be automatically mirrored depending on alliance color. Optional, defaults to true
                 this.drivebase // Requires this drive subsystem
             );
 
         }
-        
+
+        // public PPRamseteCommand rCommand(PathPlannerTrajectory path)
```

Comment:
Is this commented out code a placeholder for something that will be added later?  If not can it be removed?

Reply:
@sackihz this is your code.